src/commcare_cloud/ansible/roles/supervisor/tasks/main.yml

```
@@ -4,8 +4,19 @@
   apt: name=supervisor state=absent
   become: yes
 
+- name: Ensure pip is available
+  apt:
+    name: python3-pip
+    state: present
+  become: yes
+
 - name: install supervisor 4.0.4
-  pip: name=supervisor state=present version=4.0.4
+  pip:
+    name: supervisor
+    state: present
+    version: 4.0.4
+    executable: python3 -m pip
```

```suggestion
    executable: pip3
```
`/usr/bin/pip3` is what `python3-pip` provides, and what we use in similar instances. Is there a specific reason to change it in this case?

---

No good reason. I wanted to make sure it was using what I had just installed. But I should have checked `dpkg -L`: Seems `pip` and `pip3` would both work:

    $ dpkg -L python3-pip
    ...
    /usr/bin/pip
    /usr/bin/pip3
    ...

I'll go with your suggestion if that's what we do in other places.